client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -123,6 +137,29 @@ export const getTotalPriceBreakdown = ( state, orderId, siteId = getSelectedSite
 		: null;
 };
 
+export const getCountriesData = ( state, orderId, siteId = getSelectedSiteId( state ) ) => {
+	if ( ! isLoaded( state, orderId, siteId ) ) {
+		return null;
+	}
+
+	const shippingLabel = getShippingLabel( state, orderId, siteId );
+	const { countriesData } = shippingLabel.storeOptions;
+	if (
+		flagIsEnabled( 'woocommerce/extension-wcservices/international-labels' ) ||
+		! countriesData
+	) {
+		return countriesData;
+	}
+
+	return {
+		...pick( countriesData, [ 'PR', 'VI' ] ),
```

Comment:
Because the flag can be disabled but the plugin is updated. Once https://github.com/Automattic/woocommerce-services/pull/1382#pullrequestreview-123212672 lands, the plugin will return all the world countries, not only US/PR. So, if the plugin is updated but the flag is disabled, we need to filter down the countries to only the ones that we can reliably support without the `Customs form` feature.

We'll update the plugin first, so there will be a few hours/days when these changes will be merged into `wp-calypso` with the feature flag disabled, but the plugin will have all the REST API changes implemented.